File: SecondYear/serv-er_SarveshBaranwal_2226cseAI&ml1045_2/Week_1/whileLoop.cpp
#include<iostream>
using namespace std;

int main() {
	// Write your code here
	int n,k,even=0,odd=0;
	scanf("%d",&n);
	while(n){
		k=n%10;
		n=n/10;
		if(k%2==0){
			even+=k;
		}
		else{
			odd+=k;
		}
        
	}
    printf("%d ",even);
	printf("%d",odd);
	return 0;
}

File: SecondYear/itsrishti_SrishtiSingh_2226CSE(AI&ML)1107_2/Week_1/while_loop.java
import java.util.Scanner;

public class while_loop {	
	public static void main(String[] args) {
		// Write your code here
		Scanner scanner = new Scanner(System.in);
		int n = scanner.nextInt();
		int even = 0;
		int odd = 0;
		int temp = n;
		while(temp>0){
			int digit = temp%10;
			if(digit%2==0){
				even += digit;
			}
			else{
				odd += digit;
			}
			temp = temp/10;
		}
		System.out.print(even + " " + odd);

        scanner.close();

	}

}
